app/code/community/District/Stripe/Helper/Data.php

```diff
@@ -130,7 +130,7 @@ public function calculateCurrencyAmount($amount, $currencyCode)
             $amount = $amount * 100;
         }
 
-        return $amount;
+        return round($amount);
```

Comment:
@maximehuran Why the round?

Reply:
Because I had a case with an order of 29,61€
I need to send 2961 to Stripe and the value was 2960.9
So Stripe returned an error saying to send an integer value

Reply:
It's because on GDS we have 4 decimals after the comma.

Reply:
Ok so we can keep it and if some other websites have the same behaviour with 4 decimals, it will works. That's the required type for Stripe